src/elements/paginator/compact-paginator/compact-paginator.ts

```diff
@@ -0,0 +1,60 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import { hostAttributes } from '../../core/decorators.js';
+import { i18nPageOnTotal } from '../../core/i18n.js';
+import { SbbPaginatorCommonElementMixin } from '../common.js';
+import '../../divider.js';
+
+import style from './compact-paginator.scss?lit&inline';
+
+/**
+ * It displays a paginator component in compact mode.
+ *
+ * @event {CustomEvent<SbbPaginatorPageEventDetails>} page - Emits when the pageIndex changes.
+ */
+@customElement('sbb-compact-paginator')
+@hostAttributes({
+  role: 'group',
+})
+export class SbbCompactPaginatorElement extends SbbPaginatorCommonElementMixin(LitElement) {
+  public static override styles: CSSResultGroup = style;
+  public static readonly events: Record<string, string> = {
+    page: 'page',
+  } as const;
+
+  protected currentPageLabel(): string {
+    return i18nPageOnTotal(this.pageIndex + 1, this.numberOfPages())[this.language.current];
+  }
+
+  private _renderPageNumbers(): TemplateResult {
+    return html`
+      <span class="sbb-paginator__pages" aria-label=${this.currentPageLabel()}
```

Comment:
Other than that I'm not sure whether the aria-label competes with the aria-live region, or if they really can live together in a good way

Reply:
This was one of the point I would like to discuss because I was not sure about: i'll go with the role 'group' and then i'll wait for Mursel feedback.

The aria live region seems to work regardless of aria-label, but again, not 100% sure about correctness.

Reply:
thank you for checking it :-)

Reply:
'aria-label' attribute removed after Mursel review